code/modules/paperwork/pen.dm

```diff
@@ -47,6 +49,32 @@
 	AddElement(/datum/element/tool_renaming)
 	RegisterSignal(src, COMSIG_DART_INSERT_ADDED, PROC_REF(on_inserted_into_dart))
 	RegisterSignal(src, COMSIG_DART_INSERT_REMOVED, PROC_REF(on_removed_from_dart))
+	if (!can_click)
+		return
+	RegisterSignal(src, COMSIG_TRANSFORMING_ON_TRANSFORM, PROC_REF(on_transform))
+	if (custom_component)
+		return
```

Comment:
This var is weird, just have its own proc `setup_transforming` which subtypes can override